### PR TITLE
Fix transpose from DCVS to CCS

### PR DESCRIPTION
**Pull request by gmegh**

### CODE REVIEW

File: python/lsst/ts/imsim/closedLoopTask.py [hunk 1]
```diff
@@ -147,7 +147,7 @@ def _setSkySimBasedOnOpdFieldPos(self, instName, obsMetadata, parAngle, starMag)
         starId = 0
         raInDegArr = np.array(opdMetr.fieldX)
         decInDegArr = np.array(opdMetr.fieldY)
-        rotation = rotMatrix(obsMetadata.rotatorAngle - parAngle)
```
Comment (jbkalmbach) [inline python/lsst/ts/imsim/closedLoopTask.py:150]: Let's add a comment explaining this 180. (You can just copy and paste what I originally sent in slack).

File: doc/versionHistory.rst [hunk 2]
```diff
@@ -6,6 +6,14 @@
 Version History
 ##################
 
+.. _lsst.ts.imsim-0.5.1:
+
+-------------
+0.5.1
+-------------
+
+* Fix transpose from DCVS to CCS in `_setSkySimBasedOnOpdFieldPos()`
```
Comment (jbkalmbach) [inline doc/versionHistory.rst:15]: Correct this message to what we actually are doing now.

**Review by jbkalmbach — APPROVED** [review]
Thanks for finding this bug and fixing.

**Review by jbkalmbach — CHANGES_REQUESTED** [review]
Sorry, I was testing this change with images in the closed loop and maybe this isn't right because it's breaking the convergence. Let me take a look at this a little more.

**Review by jbkalmbach — APPROVED** [review]
Ok, the problem was some other changes I had made to try and get FAM working. This is good to go. :)

File: python/lsst/ts/imsim/closedLoopTask.py (hunk 1 above)
Comment (gmegh) [reply python/lsst/ts/imsim/closedLoopTask.py:151]: Done!

**Pull request merged**